### DISCUSSION TITLE
Remove resource from argocd without delete from k8s

### DISCUSSION THREAD

**Discussion by navry** [argoproj/argo-cd#7038] [Q&A]
Hello,

i can't find how to remove object from argocd without delete this object from k8s cluster. Is there some possibilities for that?

Thanks a lot!

Navry.

**Comment by chetan-rns** [comment]
Hi @navry, we can disable cascade to delete only the application and not the actual resources. This can be achieved from both the UI and CLI https://argoproj.github.io/argo-cd/user-guide/app_deletion/#app-deletion

**Reply by navry** [reply]
Hi @chetan-rns, thanks for answer!

This option is only for crd Application. I looking for possibility set this option only for one manifest in application. In my case, i put to argocd managing one clusterrolebiding which is managed by another tool than our argocd. So when this tool update this clusterrolebinding, our argocd see changes and i don't wont reflecting this changes in argocd too. Tries for deleting from argocd caused deleting clusterrolebinding from kubernetes but this isn't problem for us. Problem is, when we manual apply deleted clusterrolebiding back to kubernetes, argocd see this manifest too independently on previous delete by argocd ui. 

I guess argocd have to only delete some metadata from redis without calling k8s api, or i have to do it manualy.

**Reply by chetan-rns** [reply]
@navry Thanks for the details. One option can be to configure Argo CD to ignore the field that is susceptible to modification by external tools. So that the Application won't be out-of-sync.

https://argoproj.github.io/argo-cd/user-guide/diffing/#diffing-customization

**Reply by codestalkerr** [reply]
Hi @navry @chetan-rns 
Is there a solution or workaround for this introduced recently?

I am also in a situation that I need to remove a namespace file from ArgoCD but dont want to remove from kubernetes.

Please let me know if there is any way coz I tried many settings around but as soon as I try to remove the file from the repo and even if prune is disable , I still have to remove it manually from ArgoCD and there any option of deleting always deletes the resource.

**Reply by Alexander-Volk** [reply]
Any updates on this issue?

**Reply by mirdhyn** [reply]
I had the same issue: had a namespace resource that I wanted to remove from being controlled by argocd, but without actually deleting it. I've just manually removed the label `app.kubernetes.io/instance` and it disappeared from ArgoCD. 

ArgoCD actually matches resources to applications using this label.

**Comment by ragumix** [comment]
Hi!
I also stuck with such problem.
Any ideas?

**Comment by bpoole6** [comment]
You can remove the label below from your resource and then delete the manifest file. You can do this view a patch or do edit the live manifest in the Argocd UI
```
....
labels:
  argocd.argoproj.io/instance: <app_name>
```

**Comment by mickkael** [comment]
if the object is an application, here is where to change in case you have mixed up multiple applications

```
apiVersion: argoproj.io/v1alpha1
kind: Application
metadata:
  annotations:
    argocd.argoproj.io/tracking-id: <app_name>:argoproj.io/Application:argocd/<app_name>

```
